src/Configuration/ConfigurationFactory.php

```diff
@@ -65,9 +64,9 @@ class ConfigurationFactory
     private const DEFAULT_TIMEOUT = 10;
 
     private const TEST_FRAMEWORK_COVERAGE_DIRECTORY = [
-        TestFrameworkTypes::PHPUNIT => XMLLineCodeCoverage::PHP_UNIT_COVERAGE_DIR,
-        TestFrameworkTypes::PHPSPEC => XMLLineCodeCoverage::PHP_SPEC_COVERAGE_DIR,
-        TestFrameworkTypes::CODECEPTION => XMLLineCodeCoverage::CODECEPTION_COVERAGE_DIR,
+        TestFrameworkTypes::PHPUNIT => 'coverage-xml',
```

Comment:
We can't depend here on for example `PhpUnitAdapter::COVERAGE_DIR` because it will be in the separate package.

Later, we can refactor this code to return something like `$adapter::COVERAGE_DIR` - so return a dynamic variable value